### ISSUE TITLE
Bug - Shop chests opening to non-shop owners

### ISSUE THREAD

**Issue by Shanman150** [EpicEricEE/ShopChest#267]
Hey, I'm using version 2.6.0 on my 1.13.2 server, and I've found a pretty significant issue with shops. When a player crouches and sells a stack of something, with that item in their hand, they open the shop chest.

I'm not sure if this is a conflicting permission on my end (though I'm not sure what would be so specific) or a general bug with the plugin, but this is a pretty significant issue. It's been fine for my admin shops, but it's going to be pretty damaging for player shops.

**Comment by EpicEricEE** [comment]
There is no version 2.6.0 of this plugin. Are you sure you're talking about ShopChest?

**Comment by Shanman150** [comment]
You're right, I was looking at another plugin's config - areashop is what I'm using to rent out the regions. But ShopChest is the plugin I'm using for the shops themselves, I'm on one of the 1.13 snapshots. Sorry, 'bout that mix up, though the issue is still with ShopChest I think.

**Comment by Shanman150** [comment]
I have resolved this issue on my server by keeping chest access off in player shops. The admin shops still open when selling a stack of materials (right clicking with the same item as the shop is buying), but as they are admin shops, nothing is inside. For now players are having fun putting junk in the admin shops and it's not an issue. However, the bug still exists. I've simply kept it from interfering with shops on my server.

**Comment by Shanman150** [comment]
Today I had another issue with this bug. Apparently a player managed to make it work with player shops as well, and stole thousands of dollars from other players before he was caught. 

To reiterate, this bug can be duplicated by holding a stack of the item you are selling in your hand, setting the sell-hand to be the right mouse button, and crouching to sell a full stack. There might be a difference between objects and blocks, it hasn't worked consistently.

While I don't want to change our shop plugin, I can only try doing a pass on permissions on these plots once again. Players are denied the ability to open chests on player shop plots unless they are the owner of the shop, so I don't know how they could open the shop chests anyway. But shops are pretty useless if you can just take the sold item out of the chest again.

**Comment by EpicEricEE** [comment]
Please have a try with build 71 (or higher): https://ci.codemc.org/job/EpicEricEE/job/ShopChest/

**Comment by Shanman150** [comment]
Thanks! I'll give that a shot tonight.
